### ISSUE TITLE
Getting an error using `h3_h3_to_string` on a H3 cell stored in a column

### ISSUE THREAD

**Issue by mbforr** [isaacbrodsky/h3-duckdb#17]
In my instance of DuckDB this query works fine:

```sql 
select h3_h3_to_string(622236751406759935::ubigint);
```

However when I try to apply `h3_h3_to_string` to a column in my data with H3 cells stored, generated using `h3_latlng_to_cell` I get the following error:

`????INVALID VALUE - Invalid unicode (byte sequence mismatch) detected in value construction????? `

I am using DuckDB 0.7.1 and not the version that is packaged when building the H3 extension

**Comment by isaacbrodsky** [comment]
I did a quick test using some test values, and that worked:
```
D select * from 'a.parquet';
┌────────┐
│   a    │
│ uint64 │
├────────┤
│      0 │
│      1 │
│      2 │
│      3 │
│      4 │
└────────┘
D select h3_h3_to_string(a) from 'a.parquet';
┌────────────────────┐
│ h3_h3_to_string(a) │
│      varchar       │
├────────────────────┤
│ 0                  │
│ 1                  │
│ 2                  │
│ 3                  │
│ 4                  │
└────────────────────┘
```

Not reading the values from a file seems to work:
```
D select h3_h3_to_string(cast(646078419604526808 as ubigint));
┌──────────────────────────────────────────────────────┐
│ h3_h3_to_string(CAST(646078419604526808 AS UBIGINT)) │
│                       varchar                        │
├──────────────────────────────────────────────────────┤
│ 8f754e64992d6d8                                      │
└──────────────────────────────────────────────────────┘
D select cast(646078419604526808 as ubigint);
┌─────────────────────────────────────┐
│ CAST(646078419604526808 AS UBIGINT) │
│               uint64                │
├─────────────────────────────────────┤
│                  646078419604526808 │
└─────────────────────────────────────┘
```

But when my test Parquet file contains an actual H3 index, as such:
```
D select * from 'a.parquet';
┌────────────────────┐
│         a          │
│       uint64       │
├────────────────────┤
│                  0 │
│                  1 │
│                  2 │
│                  3 │
│                  4 │
│ 646078419604526808 │
└────────────────────┘
```

Then the query `select h3_h3_to_string(a) from 'a.parquet';` hangs seemingly forever. It seems to have something to do with the format string passed into StringUtils, because when I change the format string to `%lx` instead of `%llx` I get the same result you do.

**Comment by mbforr** [comment]
I had the same issue with it hanging when I read from file. The results I shared were from a table I added to a local .db file that I created from a query that used `h3_latlng_to_cell` to create the H3 cell.

**Comment by isaacbrodsky** [comment]
I was able to build a test case and (I think) fix this -- please try again with #34.
